src/SourceBuild/content/prep.sh

```diff
@@ -7,12 +7,19 @@ SCRIPT_ROOT="$(cd -P "$( dirname "$0" )" && pwd)"
 usage() {
     echo "usage: $0"
     echo ""
-    echo "  Prepares the environment to be built by downloading Private.SourceBuilt.Artifacts.*.tar.gz and"
-    echo "  installing the version of dotnet referenced in global.json"
+    echo "  Prepares the environment to be built by downloading the required archive files. This includes"
+    echo "  the previously source-built artifacts archive, prebuilts archive, and .NET SDK."
+    echo "options:"
+    echo "  --no-artifacts    Exclude the download of the previously source-built artifacts archive."
+    echo "  --no-prebuilts    Exclude the download of the prebuilts archive."
+    echo "  --no-sdk          Exclude the download of the .NET SDK."
```

Comment:
As of now, the --no-sdk option requires --no-artifacts (the bootstrap process requires an SDK).  With Logan's changes in https://github.com/dotnet/installer/pull/15603, the --no-sdk requires --no-bootstrap.  Maybe you should pull his no-bootsrap change in so that you can make them work together.